src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
+    { I18N_OPTIONS_SETTINGS26, "défilement du texte:" },
+    { I18N_OPTIONS_SETTINGS31, "haut:" },
+    { I18N_OPTIONS_SETTINGS32, "bas:" },
+    { I18N_OPTIONS_SETTINGS33, "gauche:" },
+    { I18N_OPTIONS_SETTINGS34, "droite:" },
+    { I18N_OPTIONS_SETTINGS35, "sauter:" },
+    { I18N_OPTIONS_SETTINGS36, "tirer:" },
     { I18N_OPTIONS_SETTINGS37, "grenade:" },
-    { I18N_OPTIONS_SETTINGS38, "bomb:" },
-    { I18N_OPTIONS_SETTINGS39, "frog:" },
-    { I18N_OPTIONS_SETTINGS40, "chat:" },
-    { I18N_OPTIONS_SETTINGS41, "stats:" },
-    { I18N_OPTIONS_SETTINGS42, "escape:" },
+    { I18N_OPTIONS_SETTINGS38, "bombe:" },
+    { I18N_OPTIONS_SETTINGS39, "grenouille:" },
+    { I18N_OPTIONS_SETTINGS40, "tchat:" },
+    { I18N_OPTIONS_SETTINGS41, "statistiques:" },
+    { I18N_OPTIONS_SETTINGS42, "échap:" },
     { I18N_OPTIONS_SETTINGS43, "horizontal controller deadzone:" },
     { I18N_OPTIONS_SETTINGS44, "vertical controller deadzone:" },
-    { I18N_OPTIONS_SETTINGS45, "Rescan Controllers" },
-    { I18N_OPTIONS_SETTINGS46, "Capturing" },
-    { I18N_OPTIONS_SETTINGS47, "Press key or use controller..." },
-    { I18N_OPTIONS_SETTINGS48, "${1} controller(s) found." },
-    { I18N_OPTIONS_SETTINGS49, "Controllers" },
-    { I18N_OPTIONS_SETTINGS50, "[none]" },
-    { I18N_OPTIONS_SETTINGS51, "Button: ${1}" },
+    { I18N_OPTIONS_SETTINGS45, "scanner les manettes" },
+    { I18N_OPTIONS_SETTINGS46, "capture d'écran" },
+    { I18N_OPTIONS_SETTINGS47, "Appuyer sur une touche..." },
+    { I18N_OPTIONS_SETTINGS48, "${1} manette(s) trouvée(s)." },
+    { I18N_OPTIONS_SETTINGS49, "Manettes" },
+    { I18N_OPTIONS_SETTINGS50, "[aucun]" },
+    { I18N_OPTIONS_SETTINGS51, "Bouton: ${1}" },
 
-    { I18N_CLIENT_TRANSFER, "transferring ${1} (${2}%)" },
-    { I18N_CLIENT_ENTER_MSG, "Enter Message" },
-    { I18N_CLIENT_CONNECTING_TO1, "connecting to ${1}, ping = ${2}" },
-    { I18N_CLIENT_CONNECTING_TO2, ", needs password" },
-    { I18N_CLIENT_CONNECTING_TO3, "(clients: ${1}/${2})" },
-    { I18N_CLIENT_LOGGED_IN, "you logged in" },
-    { I18N_CLIENT_LOGGED_OUT, "You logged out" },
-    { I18N_CLIENT_PING_TIMEOUT, "You logged out (ping timeout)" },
-    { I18N_CLIENT_TOO_MANY_RESENDS, "You logged out (too many resends)" },
-    { I18N_CLIENT_APPLICATION_LAYER, "You logged out (application layer quit)" },
-    { I18N_CLIENT_SERVER_MESSAGE, "Server Message" },
-    { I18N_CLIENT_MAP_INFO, "map: ${1}" },
-    { I18N_CLIENT_JOIN_REFUSED, "YOUR JOIN REQUEST WAS REFUSED" },
-    { I18N_CLIENT_FRIENDLY_FIRE, "FRIENDLY FIRE: WATCH OUT!!!" },
-    { I18N_CLIENT_UNBALANCED, "GAMEPLAY IS UNBALANCED" },
-    { I18N_CLIENT_WARMUP, "please warm up" },
-    { I18N_CLIENT_GAME_BEGINS, "game begins" },
-    { I18N_CLIENT_GAME_OVER, "GAME IS OVER" },
-    { I18N_CLIENT_RENAME, "${1} is now known as ${2}" },
-    { I18N_CLIENT_SKIN_CHANGE, "${1} changed the skin to ${2}" },
+    { I18N_CLIENT_TRANSFER, "Transferre ${1} (${2}%)" },
```

Comment:
or  "Transfert de ${1} à (${2} %)", but I don't really know the context of this thingie